implementations/GS1US.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "GS1 US",
+    "implementation": "GS1 US DID-VC-API",
+    "oauth2": {
+        "clientId": "73c9bee3-1371-409e-981f-349e8753d0fe",
+        "clientSecret": "CLIENT_SECRET_GS1US",
+        "tokenAudience": "api://f0c5b99d-e3ac-49e4-8787-bf36cabcf47f",
+        "tokenEndpoint": "https://login.microsoftonline.com/e5e65807-3d2a-4f67-a262-72f255868da0/oauth2/v2.0/token",
+        "scopes": ["api://f0c5b99d-e3ac-49e4-8787-bf36cabcf47f/.default"]
```

Comment:
This looks great, but scopes are endpoint specific so far such as here: https://github.com/w3c-ccg/vc-api-test-suite-implementations/blob/68ced5809cefe9841e4aad7e68a09df81bcd7485/implementations/Mesur.io.json#L10-L24

Are you sure your scopes are global?  The other implementations that use scopes have pretty specific scopes for each test. If you do have global scopes you could create a PR that allows them, by concat the endpoint specific scopes with the global scopes.